src/main/resources/org/jenkinsci/plugins/docker/workflow/Docker.groovy

```diff
@@ -159,8 +159,15 @@ class Docker implements Serializable {
         public void tag(String tagName = parsedId.tag, boolean force = true) {
             docker.node {
                 def taggedImageName = toQualifiedImageName(parsedId.userAndRepo + ':' + tagName)
-                // TODO as of 1.10.0 --force is deprecated; for 1.12+ do not try it even once
-                docker.script.sh "docker tag --force=${force} ${id} ${taggedImageName} || docker tag ${id} ${taggedImageName}"
+                //get docker version
```

Comment:
@jglick Probably a basic question, but the utility method you describe is on DockerClient which typically only gets created in the context of an existing Step. 

Are you advocating that 
a) he should just reuse the static method that does parsing of results 
b) we should change the scope of ```DockerClient.version()``` to static so it can be consumed easily here 
c) he should create a DockerClient object here for the purpose of determining the version? 

I'm happy to pick up the ball on these comments, just would prefer some thoughts on approach before I make a mess.